smartsim/_core/generation/generator.py

```diff
@@ -131,24 +132,26 @@ def _gen_exp_dir(self) -> None:
         else:
             logger.info("Working in previously created experiment")
 
-    def _gen_orc_dir(self, orchestrator: t.Optional[Orchestrator]) -> None:
+    def _gen_orc_dir(self, orchestrator_list: t.Optional[t.List[Orchestrator]]) -> None:
```

Comment:
It should be possible to change the type of `orchestrator_list` to `t.List[Orchestrator]` now that that is the type of `manifest.dbs`, i.e. because it cannot be `None` anymore.